src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> but we can say consistency could be an argument for reverting it.

We can make a lot of unnecessary changes for the sake of consistency. I still think I'd prefer to delete here.